svc-bip-api/src/main/java/gov/va/vro/bip/config/RabbitMqConfigProperties.java

```diff
@@ -0,0 +1,27 @@
+package gov.va.vro.bip.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+@NoArgsConstructor
+@AllArgsConstructor
+@ConfigurationProperties(prefix = "spring.rabbitmq")
+public class RabbitMqConfigProperties {
+  @Value("${spring.rabbitmq.host}")
+  private String host = "localhost";
+
+  @Value("${spring.rabbitmq.port}")
+  private int port = 5672;
+
+  @Value("${spring.rabbitmq.username}")
+  private String username = "user";
```

Comment:
I think it's probably a good idea not to include default values for sensitive data such as user ID or password in the code.

Reply:
I agree, I was under the impression this was fine, considering it's a direct copy from [MessageQueueProperties.java](https://github.com/department-of-veterans-affairs/abd-vro/blob/f768af79ad7aec19bb94b62e884dda8f6b6ebf96/shared/lib-camel-connector/src/main/java/gov/va/vro/camel/config/MessageQueueProperties.java)

Should I still remove it in that case?

Reply:
Yes, let's remove it

Reply:
Done